modules/user-access/pages/jwt-token.adoc

```diff
@@ -407,6 +420,8 @@ When using a JWT token for authentication, please consider these scenarios and h
 
 ¦ Expiration ¦ If the token has expired, an error is thrown to indicate its expiration.
 
+¦ Unsupported Features ¦ GSQL endpoints do not support JWT tokens in these versions.
```

Comment:
@Tushar-TG-14  This table is to explain the types of error messages the function can return. I don't think  you can add to this table.